File: chores.py
from twilio.rest import TwilioRestClient
import sched, time
import json

time_dict = {'d': 60, 'dishwasher': 60, 'test': 1}
with open('config.json') as data_file:
    data = json.load(data_file)

def read():
    task = input('what is the task? (d, w, dishwasher)')
    delay(int(time_dict[task]), task)
    
def send_sms(task):
    account_sid = data['twilio']['account_sid']
    auth_token = data['twilio']['auth_token']
    client = TwilioRestClient(account_sid, auth_token)
    message = ("Your " + task + " is done")
    client.messages.create(body=message, to='5163616129', from_='+15168744020')

def print_hello(task):
    print(message)

def delay(minutes, task):
    s = sched.scheduler(time.time, time.sleep)
    s.enter(minutes*60, 1, send_sms, (task,))
    s.run()

read()

File: chores.js
var twilio = require('twilio')
var schedule = require('node-schedule');
var readline = require('readline');
var config = require('./config.json')

chores = {
  'd': 60,
  'w': 40,
  'dishwasher': 60,
  'test': 1
}

var sendSms = function(task){
  var accountSid = config['twilio']['account_sid']
  var authToken = config['twilio']['auth_token']
  var client = new twilio.RestClient(accountSid, authToken)
  client.messages.create({
    body: `Your ${task} is done`,
    to: '5163616129',
    from: '+15168744020'
  }, function(err, message) {
    if (err) {
      console.log(err.message)
    }
    console.log(message.sid);
    process.exit(0);
  });
};

var read = function(){
  var rl = readline.createInterface(process.stdin, process.stdout);
  rl.setPrompt('What is the task? (d, w, dishwasher)\n');
  rl.prompt();
  rl.on('line', function(line) {
    setTimeout(sendSms, chores[line]*1000*60, line)
  });
};
read()
